### ISSUE TITLE
AVT 2 - Progress Indicator has keyboard and reflow issues

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4244]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React

## Detailed Description
Go to the React Progress Indicator (Default) Component
Click the Tab key and the Second Step has focus.
Click the Tab key again and the Third Step has focus. 
Using a mouse steps 1-4 appear as links (blue text with an underline).

Expected Result:
Based on the progress indicator usage; the user should have the ability to return to a previous step to review their data submission, so I expected steps  1-3 to be in the tab order. The invalid step should be in the tab order if it's purpose is to advise the user that there is an invalid entry. 

Zoom the screen to 400% or make the window size smaller and some of the data is no longer visible with no ability to scroll and see it. [See 1.4.10 Reflow](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html)
<img width="1375" alt="Screen Shot 2019-10-07 at 2 55 36 PM" src="https://user-images.githubusercontent.com/21676914/66344098-96c9c700-e912-11e9-85a4-8396b754a098.png">

**Comment by joshblack** [comment]
@snidersd it seems like the content is made available through tooltips, is that an appropriate way to tackle to this reflow issue? cc @dakahn

**Comment by snidersd** [comment]
@joshblack I'm not clear on the question. Are you saying the user can navigate to the step that is not visible via a tooltip? Also, here is a link to [F102 failure examples and test procedures](https://www.w3.org/WAI/WCAG21/Techniques/failures/F102) for 1.4.10 that might help answer the question.

**Comment by joshblack** [comment]
@snidersd was going off of this line:

> Zoom the screen to 400% or make the window size smaller and some of the data is no longer visible with no ability to scroll and see it.

It appears that there is a scollbar to scroll the region, and the text that is truncated has a tooltip that an individual can use to see the content.

![demo](https://user-images.githubusercontent.com/3901764/69464395-dd994200-0d43-11ea-9abd-ac3fdc4eb006.gif)

Will go through F102 now!

**Comment by snidersd** [comment]
Retested this morning and the original issue is no longer present. As noted in my original screenshot, I couldn't scroll to the left to see the first step, but it's working as expected now.
